src/components/values/OurValuesCard.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+
+export enum OurValuesIconType {
+	LEVELS = "levels",
+	IDEA = "idea",
+	THUMBSUP = "thumbsup",
+}
+
+type IconUrlMap = {
+	[key in OurValuesIconType]: string;
+};
+
+const iconSrc: IconUrlMap = {
+	levels: "/icons/levels_flag.svg",
+	idea: "/icons/idea_lightbulb.svg",
+	thumbsup: "/icons/thumbs_up.svg",
+};
+
+type OurValuesCardProps = {
+	icon: OurValuesIconType;
+	title: string;
+	description: string;
+	number: number;
+};
+
+export function OurValuesCard({
+	icon,
+	title,
+	description,
+	number,
+}: OurValuesCardProps) {
+	return (
+		<div className="relative p-8 w-96 h-80 rounded-md shadow-lg">
+			<div className="relative w-24 h-28">
+				<Image src={iconSrc[icon]} layout="fill" />
```

Comment:
Itt erdemes hozzaadni a `priority` attributumot, hogy ne lazy load-dal toltse be az svg-t.